staging/src/k8s.io/apiserver/pkg/admission/cel/metrics_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cel
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+func TestObserveDefinition(t *testing.T) {
+	defer Metrics.Reset()
+	defer legacyregistry.Reset()
+
+	Metrics.ObserveDefinition(context.TODO(), "active", "warn")
+
+	wantLabels := map[string]string{
+		"state":              "active",
+		"enforcement_action": "warn",
```

Comment:
Maybe use "deny" here since we don't have agreement in the KEP yet about anything except allow and deny?

Reply:
Gotcha, fixed